app/src/main/java/com/ruthvikbr/starbucksindiacompose/ui/components/StarbucksColumn.kt

```diff
@@ -0,0 +1,25 @@
+package com.ruthvikbr.starbucksindiacompose.ui.components
+
+import androidx.compose.foundation.background
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.unit.dp
+
+@Composable
+fun StarbucksColumn(children: @Composable () -> Unit) {
```

Comment:
@Ruthvikbr Should the children use a ColumnScope here?

Reply:
@pushpalroy The idea here was to implement a reusable column component. Hence I tried this